### PR TITLE
functions: Clarify over-time semantics

### PR DESCRIPTION
**Pull request by beorn7**
@brian-brazil

### CODE REVIEW

**Comment by matthiasr** [comment]
:+1:

File: content/docs/querying/functions.md [hunk 1]
```diff
@@ -336,3 +336,6 @@ over time and return an instant vector with per-series aggregation results:
 * `max_over_time(range-vector)`: the maximum value of all points under the specified interval.
 * `sum_over_time(range-vector)`: the sum of all values under the specified interval.
 * `count_over_time(range-vector)`: the count of all values under the specified interval.
+
+Note that all values under the specified interval have the same weight in the
```
Comment (brian-brazil) [inline content/docs/querying/functions.md]: s/under/in/ and above too.

**Comment by brian-brazil** [comment]
:+1:

File: content/docs/querying/functions.md (hunk 1 above)
Comment (beorn7) [inline content/docs/querying/functions.md]: I was already thinking… but I didn't want to doubt the native speaker who wrote this in the first place. ;)

**Pull request merged**